olp-cpp-sdk-dataservice-write/include/olp/dataservice/write/VersionedLayerClient.h

```diff
@@ -176,9 +177,20 @@ class DATASERVICE_WRITE_API VersionedLayerClient {
    * cancel operations that have been started, so you will need to call
    * CancelBatch after this if you have a batch operation in progress
    * (StartBatch successfully completed).
+   * @deprecated Use \ref CancelPendingRequests intead.
    */
+  OLP_SDK_DEPRECATED("Use CancelPendingRequests instead.")
   void CancelAll();
 
+  /**
+   * @brief Cancels all the ongoing operations that this client started.
+   *
+   * Returns instantly and does not wait for the callbacks.
+   * Use this operation to cancel all the submitted batch requests without
+   * destroying the actual client instance.
```

Comment:
Same as above. Consider mentioning that the method canceling queued and ongoing requests.